wd_helper.au3

```diff
@@ -954,7 +958,7 @@ Func _WD_ElementOptionSelect($sSession, $sStrategy, $sSelector, $sStartElement =
 		_WD_ElementAction($sSession, $sElement, 'click')
 	EndIf
 
-	Return SetError(__WD_Error($sFuncName, @error))
+	Return SetError(__WD_Error($sFuncName, @error), 0)
```

Comment:
This change shouldn't be needed.

Reply:
[_WD_ElementOptionSelect Extended removed](https://github.com/Danp2/au3WebDriver/pull/290/commits/6382e660af484518336992780ae0a5c4c4017cc7)